lib/wheel.c

```diff
@@ -123,21 +123,6 @@ void wheel_delete(struct timer_wheel *wheel)
 	XFREE(MTYPE_TIMER_WHEEL, wheel);
 }
 
-int wheel_stop(struct timer_wheel *wheel)
```

Comment:
I do not think this is a good idea to remove this code.  It provides some functionality that is not necessarily being used yet and it's in a library.